doc/constrain_R14_trend.md

```diff
@@ -0,0 +1,67 @@
+## Contrain neuton stars Equation of State (EoS) 
+
+### create an output 
+
+First of all, you need to creat an output directory, this output will host all the data that will be used to constrain the EoS.
+    
+	mkdir -p ./nmma/output
+
+
+### gennerate EoS injection.json
+
+Run this command line will genarate  a json file (injection.json)  with the BILBY processing of compact binary merging events (BNS, NSBH, optional), here it's BNS. This injection contents a simulation of the parameters : luminosity_distance, log10_mej_wind, KNphi, inclination_EM, KNtimeshift, geocent_time.
+At the end of the run, when the process is ok,  an injection.json will be find at  ./nmma/output directory.
+
+	nmma_create_injection --prior-file ./nmma/example_files/prior/ZTF_kn.prior --eos-file ./nmma/example_files/eos/ALF2.dat --binary-type BNS -f ./nmma/output/injection --n-injection 100 --original-parameters  --extension json
```

Comment:
Let's use the prior files in priors:

nmma_create_injection --prior-file ./nmma/priors/Bu2019lm.prior --eos-file ./nmma/example_files/eos/ALF2.dat --binary-type BNS -f ./nmma/output/injection --n-injection 100 --original-parameters  --extension json